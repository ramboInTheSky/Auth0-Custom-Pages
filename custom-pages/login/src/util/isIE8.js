export const isIE8 = () => {
  if (navigator.appName.indexOf('Internet Explorer') !== -1) {
    // yeah, he's using IE
    const badBrowser =
      navigator.appVersion.indexOf('MSIE 9') === -1 && // v9 is ok
      navigator.appVersion.indexOf('MSIE 1') === -1 // v10, 11, 12, etc. is fine too

    if (badBrowser) {
      return true
    }
    return false
  }
  return false
}

export default isIE8
