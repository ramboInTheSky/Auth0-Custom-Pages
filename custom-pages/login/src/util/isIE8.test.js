import isIE8 from './isIE8'


describe('AppContainer components', () => {
  it('should create an instance of isIE8 correctly', () => {
    expect(isIE8).toMatchSnapshot()
  })
})
