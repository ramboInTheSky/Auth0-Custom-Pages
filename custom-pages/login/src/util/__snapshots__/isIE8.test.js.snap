// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppContainer components should create an instance of isIE8 correctly 1`] = `[Function]`;
